.travis.yml

```diff
@@ -17,6 +18,7 @@ services:
 
 before_script:
   - psql -c 'create database trellis;' -U postgres
+  - psql -c 'create database trellisMP;' -U postgres
```

Comment:
Just wondering about why you are creating this separate DB and not reusing the trellis one.

Reply:
This is just me experimenting with configuration options. Re-using the same database leads to problems b/c I have two different database "version" managers (Liquibase for the dropwizard system and Flyway for the quarkus system), and trying to use both is problematic. A lot of these recent commits (as you will note) are just trial and error -- the tests all pass fine locally, but getting it to work on a remote CI system can be more complicated.

Reply:
FYI, I'm no longer creating a separate database. Instead, I use the travis-managed postgresql database as part of a separate build job.